backend/package.json

```diff
@@ -5,7 +5,7 @@
   "private": true,
   "license": "Affero GPL 3.0",
   "scripts": {
-    "build": "nest build",
+    "build": "nest build && docker build -t my-code-runner-python ./src/runner/dockers/python",
```

Comment:
lets move docker build command from `backend/package.json` into main `Makefile` (or backend/Makefile). 
We use Makefile specifically for infrastructure/project preparation management